src/api.py

```diff
@@ -1422,11 +1424,25 @@ def HandleAdd(self, a, b):
         """Test two numeric params"""
         return a + b
 
+    @command('clearUISignalQueue')
```

Comment:
something to make this only available in test mode

Reply:
My preferred way would be to create an additional decorator for testmode functions.